### PR TITLE
Improve Logout URL method

### PR DESCRIPTION
**Pull request by alexnoox**
This PR fix the logout url to be redirected to when logging out an app.

@x4d3 Can you review?

### CODE REVIEW

**Comment by alexnoox** [comment]
Implementation of this feature in the demoapp-php: https://github.com/maestrano/demoapp-php/pull/3

File: lib/Sso/Service.php [hunk 1]
```diff
@@ -104,6 +106,19 @@ public function getLogoutUrl() {
   }
 
   /**
+
+   * Return where the app should redirect after logging user out
+   *
+   * @return string url
+   */
+  public function getLogoutUrlWithUserUid($userUid) {
```
Comment (x4d3) [inline lib/Sso/Service.php:114]: I see that method overloading doe snot exist in php...
http://stackoverflow.com/questions/4697705/php-function-overloading

File: lib/Sso/Session.php [hunk 2]
```diff
@@ -135,6 +135,14 @@ public function performRemoteCheck($httpClient = null) {
       return false;
     }
 
+    /**
+     * @return the Idp logout url where users should be redirected to upon logging out
+     */
+    public function getLogoutUrl()
+    {
+      return Maestrano::with($this->_preset)->sso()->getLogoutUrlWithUserUid($this->uid);
```
Comment (x4d3) [inline lib/Sso/Session.php:143]: would it be possible to assign Maestrano::with($this->_preset) to a variable in the whole file ? ( I suppose more refactoring is needed...)

**Review by x4d3 — APPROVED** [review]
I approve

**Pull request merged**